src/app/[locale]/(routes)/loans/loans.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC } from "react";
+import { _TLoan } from "@/models/loan";
+import { deleteLoanAction } from "@/app/_action";
+
+type Props = {
+  loans: _TLoan[];
+  results: number;
+};
+const Loans: FC<Props> = ({ loans, results }) => {
+  async function action(data: FormData) {
+    "use server";
+
+    const id = data.get("id");
+    if (!id || typeof id !== "string") {
+      return;
+    }
+
+    // Invoke server action to add new loan
+    await deleteLoanAction({ id, path: "/loans" });
+  }
+  return (
+    <section className="space-y-16">
+      <p className="my-8">Results: {results}</p>
+
+      {loans.map((loan) => (
+        <dl
+          key={loan.id}
+          className="grid grid-cols-2 md:grid-cols-4 place-items-start text-sm sm:text-base [&>*]:w-full [&>*]:border-b-2 [&>*]:border-dotted [&>*]:py-2 [&>*]:max-w-sm [&>*]:truncate gap-x-4"
+        >
+          <dt>ID:</dt>
+          <dd>{loan.id}</dd>
+          <dt>Borrower Address:</dt>
```

Comment:
We are using 18n, can not be hardcoded strings

Reply:
i18n not configure in main check in your end